packages/mds-transactions-service/tests/transaction-operations.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { uuid } from '@mds-core/mds-utils'
+import { TransactionServiceManager } from '../service/manager'
+import { TransactionServiceClient } from '../client'
+import { TransactionRepository } from '../repository'
+import { TransactionOperationDomainCreateModel } from '../@types'
+
+function* transactionOperationsGenerator(length = 20): Generator<TransactionOperationDomainCreateModel> {
+  const start_timestamp = Date.now() - length * 1000
+
+  for (let i = 0; i < length; i++) {
+    const timestamp = start_timestamp + i * 1000
+
+    yield {
+      transaction_id: uuid(),
+      operation_id: uuid(),
+      timestamp,
+      operation_type: 'invoice_generated',
+      author: 'no one'
+    }
+  }
+}
+
+const TransactionServer = TransactionServiceManager.controller()
+
+describe('Transaction Operation Tests', () => {
+  beforeAll(async () => {
+    await TransactionServer.start()
+  })
+
+  /**
+   * Clear DB after each test runs. No side-effects for you.
```

Comment:
This assumes DB is "empty" initially. `beforeEach/afterAll` would be certain.

Reply:
Resolved in 2bc23236f78585cb9f24ea4ed807c4659a4aa2b5